### DISCUSSION TITLE
Any definitions for power numbers in the docs?

### DISCUSSION THREAD

**Discussion by dalyles** [The-OpenROAD-Project/OpenROAD-flow-scripts#1845] [Q&A]
Hello,

I also wanted to know if there is a clear definition for the words <code>Internal</code>, <code>Switching</code>, and <code>Leakage</code> power so that we won't run into any ambiguities while analyzing power numbers.

Please let me know if anyone can clarify.

Thanks

**Comment by maliberty** [comment]
I believe these are standard terms defined by the Liberty format.  Roughly

Leakage - power when non-switching
Internal - power when switching related to the internals of the cell
Switching - power when switching related to the load of the cell

**Reply by dalyles** [reply]
Can you please elaborate on the "internals" part?
Thanks!

**Reply by rovinski** [reply]
https://blogs.cuit.columbia.edu/zp2130/modeling_power_terminology/
https://asic-soc.blogspot.com/2008/04/dynamic-and-internal-power.html
